scipy/special/_logsumexp.py

```diff
@@ -58,7 +60,8 @@ def logsumexp(a, axis=None, b=None, keepdims=False, return_sign=False):
 
     See Also
     --------
-    numpy.logaddexp, numpy.logaddexp2
+    :data:`numpy.logaddexp`
+    :data:`numpy.logaddexp2`
```

Comment:
Work around to 
- https://github.com/numpy/numpy/issues/28734
- https://github.com/numpy/numpydoc/issues/618

Note that docscrape has been bumped up recently-ish: https://github.com/scipy/scipy/pull/21215